virtualenv.py

```diff
@@ -904,6 +905,34 @@ def space_path2url(p):
         logger.end_progress()
 
 
+def create_proxies(py_executable, bin_dir):
+    scripts = glob.glob(os.path.join(bin_dir, "pip*"))
+    scripts += glob.glob(os.path.join(bin_dir, "easy_install*"))
+    scripts += glob.glob(os.path.join(bin_dir, "wheel*"))
+
+    execute_bits = stat.S_IEXEC | stat.S_IXGRP | stat.S_IXOTH
+
+    for script in scripts:
+        head, tail = os.path.split(script)
+        hidden_destination = os.path.join(head, '.' + tail)
+
+        os.rename(script, hidden_destination)
+
+        with open(script, "w") as script_proxy:
+            text = "#!/usr/bin/env sh"
```

Comment:
Should this not force an appropriate encoding? What if the venv name contains non-ASCII characters?